src/unix/fs.c

```diff
@@ -1177,41 +1100,18 @@ static ssize_t uv__fs_write(uv_fs_t* req) {
     abort();
 #endif
```

Comment:
Aside: this was a bug fix for a really old version of macos (you know, back when apple still called it "os x".)

I _think_ it can be removed but I don't really have the hardware anymore to test.

Reply:
I have an old mac. Do we have a test that covers this case?

Reply:
We don't but node does: test/parallel/test-fs-sir-writes-alot.js - and it's pretty much the same test it was back then.

Reply:
FWIW, in my `10.15.7` mac, that test passes after removing the locks